src/adaptive/index.es

```diff
@@ -26,9 +27,16 @@ const makeAdaptive = (Component) => (
 
     render() {
       const { isMobile } = this.state;
-      return <Component {...this.props} mobile={isMobile} />;
+      const { forwardedRef, ...props } = this.props;
```

Comment:
add `forwardRef` to `propTypes`?

Reply:
actually there's no propTypes validation, build script strips that out anyway.